edx_api_doc_tools/data.py

```diff
@@ -0,0 +1,100 @@
+"""
+Data definitions for API schemas.
+
+These are generally thin wrappers around drf-yasg's
+OpenAPI schema types. We wrap them so that we can
+easily switch from drf-yasg to another supporting
+library if we must without affecting clients of this
+library.
```

Comment:
Why do you wrap docstrings and comments so narrowly?

Reply:
Bad habit, or an overreaction to seeing 120+-wide unwrapped docstrings, I guess. I'll try hard to wrap at column ~80.